assets/src/styles/blocks/OldCovers/TakeActionBoxoutEditor.scss

```diff
@@ -1,20 +1,17 @@
 .cover-card.action-card {
   position: relative;
-  max-width: 33%;
-  margin-right: 0;
-  margin-left: auto;
 
   a {
     pointer-events: none;
   }
 
-  html[dir="rtl"] & {
-    margin-left: 0;
-    margin-right: auto;
-  }
-
   .cover-card-excerpt,
   .cover-card-btn {
     display: block;
   }
+
```

Comment:
Don't we want to remove everything from the `OldCovers` folder? 🤔

Reply:
It also seems it's [not even used anymore](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/715/files#diff-1d8227442d5b198c6d5362b74a30d9a58a34c4de87df6535394bdddec936267cL23)

Reply:
Checking, it's actually supposed to be there, because we need the fix added below :thinking: Maybe merging/rebasing went wrong.

Reply:
Ah it was a fix from before I had to change the class names, good catch. Moved it to the right directory and loaded it in the editor.